pom.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright © 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.gravitee.policy</groupId>
+    <artifactId>gravitee-policy-http-redirect</artifactId>
+    <version>1.0.0</version>
+
+    <name>Gravitee.io APIM - Policy - HTTP Request Redirect</name>
+    <description>Redirects client HTTP requests to different locations using an HTTP redirect response</description>
+
+    <parent>
+        <groupId>io.gravitee</groupId>
+        <artifactId>gravitee-parent</artifactId>
+        <version>22.2.4</version>
+    </parent>
+
+    <properties>
+            <gravitee-apim.version>4.6.0</gravitee-apim.version>
```

Comment:
Since it's a new policy, why not release it with 4.7 or even 4.8 for and and use the java 21 parent directly. that'll be one less to migrate later, won't it?

Reply:
oh you are right thanks